src/app/features/activity-list/activity-list.tsx

```diff
@@ -11,14 +11,26 @@ import { SubmittedTransactionList } from './components/submitted-transaction-lis
 import { TransactionList } from './components/transaction-list/transaction-list';
 import { PendingTransactionList } from './components/pending-transaction-list/pending-transaction-list';
 import { convertBitcoinTxsToListType, convertStacksTxsToListType } from './activity-list.utils';
+import { LoadingSpinner } from '@app/components/loading-spinner';
```

Comment:
I try to keep aliased imports grouped together above local directory imports if possible.